app/src/modules/theme/index.ts

```diff
@@ -0,0 +1,56 @@
+import { createSlice } from '@reduxjs/toolkit';
+import storage from 'utils/storage'
+
+export const themes = ['light' , 'dark'] as const
+export type Theme = typeof themes[number];
+
+interface ThemeState {
+  theme: Theme
+};
+
+const initialState: ThemeState = { theme: initialLoadTheme() };
+
+export const slice = createSlice({
+  name: 'selectedScreen',
+  initialState,
+  reducers: {
+    changeTheme: (state, action) => {
+      const theme = action.payload
+      storage.set('theme', theme)
```

Comment:
No need to deal with storage here. Because redux is automatically stored in local storage here -> https://github.com/RStankov/FocusedTask/blob/master/app/src/modules/index.tsx#L16